packages/esm-patient-chart-app/src/config-schema.ts

```diff
@@ -16,6 +16,11 @@ export const esmPatientChartSchema = {
     _default: false,
     _description: 'Disable notes/tests/medications/encounters tabs when empty',
   },
+  drugOrderTypeConceptUuid: {
```

Comment:
Can we use the config from the medications app?

```ts
const { drugOrderTypeUUID } = useConfig({ externalModuleName: '@openmrs/esm-patient-medications-app' })

```

Reply:
Yeah, makes sense to have a single source of truth